course-definition.yml

```diff
@@ -676,4 +687,96 @@ stages:
     primary_extension_slug: "pub-sub"
     name: "Unsubscribe"
     difficulty: medium
-    marketing_md: In this stage, you'll add support for the `UNSUBSCRIBE command`, which is used to unsubscribe from a channel.
\ No newline at end of file
+    marketing_md: In this stage, you'll add support for the `UNSUBSCRIBE command`, which is used to unsubscribe from a channel.
+
+  # Sorted Sets
+  - slug: "ct1"
+    primary_extension_slug: "zsets"
+    name: "Create sorted set"
+    difficulty: easy
+    marketing_md: |
+      In this stage, you'll add support for creating a new sorted set using the `ZADD` command.
+
+  - slug: "hl2"
+    primary_extension_slug: "zsets"
+    name: "Validate score values"
+    difficulty: easy
+    marketing_md: |
+      In this stage, you'll add support for verifying the value of the score argument for `ZADD` command.
+
+  - slug: "hf1"
+    primary_extension_slug: "zsets"
+    name: "Add to existing zset"
+    difficulty: easy
+    marketing_md: |
+      In this stage, you'll add support for adding elements to an existing zset.
+
+  - slug: "kn4"
+    primary_extension_slug: "zsets"
+    name: "Count zset members"
+    difficulty: easy
+    marketing_md: |
+      In this stage, you'll add support for counting the number of members in a zset.
+
+  - slug: "gd7"
+    primary_extension_slug: "zsets"
+    name: "Retrieve member score"
+    difficulty: easy
+    marketing_md: |
+      In this stage, you'll add support for retrieving the score of a zset member.
+
+  - slug: "lg6"
+    primary_extension_slug: "zsets"
+    name: "Retrieve member rank"
+    difficulty: medium
+    marketing_md: |
+      In this stage, you'll add support for retrieving the rank of a zset member.
+
+  - slug: "ic1"
+    primary_extension_slug: "zsets"
+    name: "List zset members"
+    difficulty: medium
+    marketing_md: |
+      In this stage, you'll add support for listing the zset members using the `ZRANGE` command.
+
+  - slug: "bj4"
+    primary_extension_slug: "zsets"
+    name: "ZRANGE with negative indexes"
+    difficulty: easy
+    marketing_md: |
+      In this stage, you'll add support for negative indexes for the `ZRANGE` command.
+
+  - slug: "zu9"
+    primary_extension_slug: "zsets"
+    name: "List members in reverse"
+    difficulty: easy
+    marketing_md: |
+      In this stage, you'll add support for `REV` option for the `ZRANGE` command.
+
+  - slug: "sq7"
```

Comment:
Let's avoid these remove commands too, to keep things simple - `ZREM` is likely to be more popular than methods like `ZPOPMIN` so it'd be weird to implement this and not `ZREM`. Also looks like we added this to get to the blocking commands which we're not doing anymore so could be a weird ending to the extension. 

We could add `ZREM` instead. Reasons: (a) it's a more popular command (b) it's probably going to exercise the user's data structure more since it removes from the middle of the list and not just start / end.